src/main/java/de/retest/recheck/ui/diff/meta/MetadataDifferenceFinder.java

```diff
@@ -10,24 +10,24 @@
 
 public class MetadataDifferenceFinder {
 
-	public MetadataDifference findDifferences( final SutState expected, final SutState actual ) {
-		return findDifferences( expected.getMetadata(), actual.getMetadata() );
+	public MetadataDifference findDifferences( final SutState actual, final SutState expected ) {
```

Comment:
Everywhere else, it is `expected, actual` ... I would keep that order to be consistent.

Reply:
Well, that's embarrassing... 
`RecheckDifferenceFinder#findDifferences` is changed, therefore I assumed this was mixed, too.

Will revert that

Reply:
Oh. We should fix `RecheckDifferenceFinder#findDifferences` as well...

Reply:
Done